test/DynamoCoreTests/LibraryTests.cs

```diff
@@ -232,6 +233,8 @@ public void MethodWithRefOutParams_NoLoad()
         [Category("UnitTests")]
         public void DumpLibraryToXmlZeroTouchTest()
         {
+            var searchViewModel = new SearchViewModel(null, new NodeSearchModel());
+
```

Comment:
Good point. Now the `LibraryTest` no longer inherits from `DSEvaluationViewModelUnitTest`